client/datatypes.go

```diff
@@ -2,9 +2,9 @@
 // User Types
 //
 // Generated with goagen v0.2.dev, command line:
-// $ goagen
+// $ goagen.exe
```

Comment:
I would gladly do so if we had a clean separation of what is generated 
and what is not, but we don't, as we DO edit generated files.

On a more general, I am opposed to having commits that would

a) lead to a project that does not compile
b) take stuff apart that is not really separate, like adding something 
to the design and regenerating the dependent files.

/Thomas

Am 24.06.2016 um 08:07 schrieb Konrad Kleine:
>
> In client/datatypes.go 
> <https://github.com/almighty/almighty-core/pull/4#discussion_r68357116>:
>
> > @@ -2,9 +2,9 @@
> >  // User Types
> >  //
> >  // Generated with goagen v0.2.dev, command line:
> > -// $ goagen
> > +// $ goagen.exe
>
> Can you please separate the generated code from the written code in 
> commits?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/almighty/almighty-core/pull/4/files/6027d22ed35050dfc95fdf2ea1abf15d3763e94a..90e1426b9cf7e73b8bbed3b7aa447293d855f203#r68357116>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AMjc-rGBpM03HU_Mq4uQXOw0eX7LxXlGks5qO3QTgaJpZM4I8524>.
>